src/tif_model_iterator/tif_kernel_iterator.py

```diff
@@ -28,7 +26,7 @@
 """
 
 
-class tif_kernel_iterator_generator:
+class TifKernelIteratorGenerator:
 
     """
     This class set up a .tif image in order to easily extracts kernel from it.
```

Comment:
Still need to update documentation for new code

Reply:
Correct! Good point!